rotkehlchen/globaldb/schema.py

```diff
@@ -303,6 +303,17 @@
 );
 """
 
+# Used to store exchange specific asset's symbol mappings with their identifiers.
+# location column can be NULL, which means the asset mapping is common for all the exchanges.
+DB_CREATE_LOCATION_ASSET_MAPPINGS = """
+CREATE TABLE IF NOT EXISTS location_asset_mappings (
+    location TEXT,
+    value TEXT NOT NULL,
```

Comment:
can you change it and be explicit? Instead of value use `exchange_symbol`. It is easier to follow than `value`